lib/copy-sync/__tests__/copy-sync-dir.test.js

```diff
@@ -27,15 +27,15 @@ describe('+ copySync()', () => {
       src = path.join(TEST_DIR, 'src')
       dest = path.join(TEST_DIR, 'dest')
 
-      fs.mkdirsSync(src)
+      fs.mkdirSync(src)
```

Comment:
> And also sorry that I am still not in harmony with you guys. I try my best to be as helpful as possible.

@manidlou you're very helpful, thank you!!

> @jprichardson Perhaps we should formalize some collaborator guidelines sometime?

I think for now, any modifications (including my own), barring some crazy necessary hot-fix, require at least one other reviewer? 

To be clear, it's okay to accept a self-submitted PR, but it must be for a very good reason. We're all professionals here, so I think this loose guideline should be acceptable for now.

Sound good @manidlou @RyanZim?